animation_nodes/nodes/material/material_output.py

```diff
@@ -38,7 +38,7 @@ def getExecutionCode(self, required):
             if self.inputs[i].isUsed: yield f"    self.{setFunction}(material, '{attr}', {attr})"
 
     def setAttribute_Color(self, material, attribute, color):
-        newColor = color[:3]
+        newColor = color[:4]
```

Comment:
You can just removed this line and use the `color` variable directly instead of `newColor`.

Reply:
Done in 6083254059e18a91aa0c39ad4e6eea9e5e53da36